src/plugins/vis_type_metric/public/metric_vis_type.ts

```diff
@@ -29,6 +29,7 @@ export const createMetricVisTypeDefinition = (): VisTypeDefinition<VisParams> =>
       type: 'metric',
       metric: {
         percentageMode: false,
+        percentageFormatPattern: undefined,
```

Comment:
nit: what is a necessity to specify `undefined` explicitly?

Reply:
Removed it